### PR TITLE
Development 313 login UI color

### PR DESCRIPTION
**Pull request by ehab-hassan**
### Related Issues
#1323

### Changes
- changed the theme to match the new playground.
- moved (close button) to the right of the Login button.

image
image
image

### CODE REVIEW

File: packages/playground/src/components/weblet_layout.vue [hunk 1]
```diff
@@ -21,7 +21,7 @@
       </div>
     </section>
 
-    <v-divider :class="{ 'mb-2': true, 'mt-5': !!$slots.subtitle, 'mt-2': !$slots.subtitle }" />
+    <!-- <v-divider :class="{ 'mb-2': true, 'mt-5': !!$slots.subtitle, 'mt-2': !$slots.subtitle }" /> -->
```
Comment (amiraabouhadid) [inline packages/playground/src/components/weblet_layout.vue:24]: please remove commented out code

File: packages/playground/src/weblets/profile_manager.vue [hunk 2]
```diff
@@ -62,16 +62,28 @@
...
       </VCard>
     </template>
-
+    <v-card color="primary" class="d-flex justify-center items-center mt-3 pa-3 text-center">
+      <v-card-title class="pa-0">Connect your TFChain Wallet</v-card-title>
+    </v-card>
     <WebletLayout disable-alerts>
-      <template #title> Connect your TFChain Wallet </template>
-      <template #subtitle>
+      <!-- <template #title> Connect your TFChain Wallet </template> -->
```
Comment (amiraabouhadid) [inline packages/playground/src/weblets/profile_manager.vue:69]: please remove this as well

File: packages/playground/src/weblets/profile_manager.vue [hunk 3]
```diff
@@ -62,16 +62,28 @@
...
+        <p :style="{ maxWidth: '880px' }">
+          Please visit
+          <a class="app-link" href="https://manual.grid.tf/playground/wallet_connector.html" target="_blank">
+            the manual
+          </a>
+          get started.
+        </p>
+      </v-alert>
+
+      <!-- <template #subtitle>
```
Comment (amiraabouhadid) [inline packages/playground/src/weblets/profile_manager.vue:80]: here too

**Review by amiraabouhadid — CHANGES_REQUESTED** [review]

File: packages/playground/src/weblets/profile_manager.vue [hunk 4]
```diff
@@ -62,17 +62,21 @@
...
-      <template #title> Connect your TFChain Wallet </template>
-      <template #subtitle>
-        Please visit
-        <a class="app-link" href="https://manual.grid.tf/playground/wallet_connector.html" target="_blank">
-          the manual
-        </a>
-        get started.
-      </template>
-
+      <!-- <template #title> Connect your TFChain Wallet </template> -->
```
Comment (mohamedamer453) [inline packages/playground/src/weblets/profile_manager.vue:69]: A commented line still needs to be removed

**Review by mohamedamer453 — APPROVED** [review]

**Review by zaelgohary — COMMENTED** [review]
Form validation works once I click on connect tab.

image

File: packages/playground/src/weblets/profile_manager.vue [hunk 5]
```diff
@@ -245,112 +249,120 @@
...
+            <CopyInputWrapper :data="profileManager.profile.address" #="{ props }">
+              <VTextField label="Address" readonly v-model="profileManager.profile.address" v-bind="props" />
+            </CopyInputWrapper>
+          </v-col>
+          <v-divider class="mx-4" vertical></v-divider>
+
+          <v-col>
+            <section class="d-flex flex-column align-center">
+              <p class="mb-4 text-center">
+                Scan the QRcode using
```
Comment (zaelgohary) [inline packages/playground/src/weblets/profile_manager.vue:337]:
```suggestion
                Scan the QR code using
```

**Review by zaelgohary — CHANGES_REQUESTED** [review]

**Comment by zaelgohary** [comment]
> Form validation works once I click on connect tab.
> 
> image

Will be discarded as this PR handles the UI changes only.

**Review by zaelgohary — APPROVED** [review]

**Review by amiraabouhadid — APPROVED** [review]

**Pull request merged**